src/main/java/easytests/api/v1/controllers/QuestionsController.java

```diff
@@ -34,11 +46,31 @@
     /**
      * update
      */
-    /**
-     * show(questionId)
-     */
+    @GetMapping("/{questionId}")
+    public Question show(@PathVariable Integer questionId) throws NotFoundException, ForbiddenException {
+        final QuestionModelInterface questionModel = this.getQuestionModel(
+                questionId,
+                (new QuestionsOptions()).withAnswers(new AnswersOptions())
```

Comment:
I would prefere to use `this.questionsOptionsBuilder.forAuth()` and than add withAnswers.